### PR TITLE
add the powerPreference for CreateContext for node binding

### PR DESCRIPTION
**Pull request by lionkunonly**
Add the `powerPreference` option for Context. It resolves the two failed cases [failed case 1](https://github.com/webmachinelearning/webnn-polyfill/blob/7d33b3c62918cd4dab1753446086c9d3261b691a/test/api/context.js#L36)  and [failed case 2](https://github.com/webmachinelearning/webnn-polyfill/blob/7d33b3c62918cd4dab1753446086c9d3261b691a/test/api/context.js#L40).

### CODE REVIEW

File: node/src/Context.cpp [hunk 1]
```diff
@@ -23,9 +24,29 @@ Napi::FunctionReference node::Context::constructor;
 namespace node {
 
     Context::Context(const Napi::CallbackInfo& info) : Napi::ObjectWrap<Context>(info) {
+        MLContextOptions options;
+        if (info.Length() > 0) {
+            Napi::Object optionsObject = info[0].As<Napi::Object>();
+            if (optionsObject.Has("powerPreference")) {
```
Comment (fujunwei) [inline node/src/Context.cpp:30]: Please add `devicePreference`.

File: node/src/ML.cpp [hunk 2]
```diff
@@ -14,7 +14,9 @@
 
 #include "ML.h"
 
+#include <iostream>
```
Comment (fujunwei) [inline node/src/ML.cpp:17]: Remove this line.

File: node/src/ML.cpp [hunk 3]
```diff
@@ -24,7 +26,18 @@ namespace node {
     }
 
     Napi::Value ML::CreateContext(const Napi::CallbackInfo& info) {
-        Napi::Object context = Context::constructor.New({});
+        // std::cout<<info.Length()<<std::endl;
```
Comment (fujunwei) [inline node/src/ML.cpp:29]: Ditto.

File: node/src/ML.cpp [hunk 4]
```diff
@@ -24,7 +26,18 @@ namespace node {
...
+        WEBNN_NODE_ASSERT(info.Length() <= 1, "The number of arguments is invalid.");
+
+        Napi::Object context;
```
Comment (fujunwei) [inline node/src/ML.cpp:32]:
```
std::vector<napi_value> args = {};
if (info.Length() > 0) args.push_back(info[0].As<Napi::Object>());
return Context::constructor.New(args);
```

File: node/src/Context.cpp (hunk 1 above)
Comment (lionkunonly) [reply node/src/Context.cpp:30]: I will add it in the next PR

File: node/src/ML.cpp (hunk 4 above)
Comment (lionkunonly) [reply node/src/ML.cpp:30]: I do not think this implementation check if the parameter is a Object. So I use a similar way in the next commit. Please check it.

File: node/src/ML.cpp (hunk 3 above)
Comment (lionkunonly) [reply node/src/ML.cpp]: Done

File: node/src/ML.cpp (hunk 2 above)
Comment (lionkunonly) [reply node/src/ML.cpp]: Done.

**Comment by huningxin** [comment]
@fujunwei , please take another look as @lionkunonly updated the PR.

**Review by fujunwei — APPROVED** [review]
LGTM.

**Pull request merged**